src/app/admin/volunteers/volunteers-details/volunteers-details.component.ts

```diff
@@ -21,12 +29,27 @@ export class VolunteersDetailsComponent implements OnInit, OnDestroy {
     private route: ActivatedRoute,
     private volunteerFacade: VolunteersFacade,
     private translateService: TranslateService,
-  ) {}
+  ) {
+    this.route.paramMap
+      .pipe(
+        map((params) => params.get('id') as string),
+        tap((id) => (this.recordId = id)),
+        takeUntil(this._destroy),
+      )
+      .subscribe((id) => {
+        this.recordId = id;
+        if (id) {
+          this.volunteerFacade.getVolunteerById(id);
```

Comment:
`this.volunteer$` is already loaded and passed inside component by router see

- src/app/admin/volunteers/shared/volunteer.resolver.ts
- src/app/admin/volunteers/volunteers.routes.ts:23

I supose this call is just to populate redux `details` state